openshift_templates/collectd-config.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: collectd-config
+data:
+  host: " " 
```

Comment:
Why are these one space? Could they be empty string?

Reply:
Yes, they can be a empty string. Fixed.